analyzer/library/pip/testdata/requirements.txt

```diff
@@ -0,0 +1,6 @@
+click==8.0.0
+Flask==2.0.0
+itsdangerous==2.0.0
+Jinja2==3.0.0
+MarkupSafe==2.0.0
+Werkzeug==2.0.0
```

Comment:
Could you add more specifiers other than `==`?

Reply:
I can add, but it will test logic in another repository
https://github.com/aquasecurity/go-dep-parser/tree/main/pkg/pip
which as far I can see has some test coverage

Reply:
Yes, you don't have to add all edge cases. I just want to add common `requirements.txt` including `==` and some other specifiers.

Reply:
fixed. As far I can see everything but `==` is ignored